images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -861,7 +864,9 @@ do
   SERVICE_NAME_IMAGE="${MAP_SERVICE_NAME_TO_IMAGENAME[${SERVICE_NAME}]}"
   SERVICE_NAME_IMAGE_HASH="${IMAGE_HASHES[${SERVICE_NAME_IMAGE}]}"
 
-  helm template ${SERVICE_NAME} /kubectl-build-deploy/helmcharts/${SERVICE_TYPE} -f /kubectl-build-deploy/values.yaml --set image="${SERVICE_NAME_IMAGE_HASH}" | outputToYaml
+  if [ "${SERVICE_TYPE}" != "mariadb-shared" ]; then
```

Comment:
let's remove this `if`. as the helm chart should just create the exact same CRD again, plus the helmchart might have additional objects in it that are not added yet (like `prebackuppods`)